daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/TransactionCoder.scala

```diff
@@ -449,6 +451,18 @@ object TransactionCoder {
     val nodeId = decodeNid.fromString(protoNode.getNodeId)
 
     protoNode.getNodeTypeCase match {
+      case NodeTypeCase.ROLLBACK =>
+        val protoRollback = protoNode.getRollback
+        val childrenOrError = protoRollback.getChildrenList.asScala
+          .foldLeft[Either[DecodeError, BackStack[Nid]]](Right(BackStack.empty[Nid])) {
+            case (Left(e), _) => Left(e)
+            case (Right(ids), s) => decodeNid.fromString(s).map(ids :+ _)
+          }
```

Comment:
Maybe worth factoring out? Seems very similar to the one to decode the children of exercise nodes.

Reply:
ok, done